src/actions/savedAccounts.js

```diff
@@ -1,12 +1,19 @@
 import actionTypes from '../constants/actions';
-import { getSavedAccount, setSavedAccount, removeSavedAccount } from '../utils/saveAccount';
+import {
+  getSavedAccounts,
+  setSavedAccount,
+  removeSavedAccount,
+  setLastActiveAccount,
+  getLastActiveAccount,
+} from '../utils/savedAccounts';
 
 /**
  * An action to dispatch accountSaved
  *
  */
 export const accountSaved = (data) => {
   setSavedAccount(data);
```

Comment:
I really like your choice of calling this utilis here.

Reply:
LOL https://github.com/LiskHQ/lisk-nano/blame/4170b7b1836ddd3b5ae61d1912b34702c7a0d2dc/src/actions/savedAccounts.js